ParisBNF/abb/BNFabb29.xml

```diff
@@ -335,13 +331,11 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                             <msPart xml:id="p2">
                         <msIdentifier/>
                         <msContents>
-                           <summary/>
```

Comment:
I am always in favour of empty `summary `elements, this will allow the app to print the summary

Reply:
If you look at the description now, what are the indications "extracted" and "given", under "Codicological Unit p1"
"Summary of codicological unit"? I dound them distracting. My intention was to remove them.

Reply:
it is displayed correctly in the right column when there is history after physDesc (I have to explore why, but you can add a history element maybe)

Reply:
mss?

Reply:
I think it is very helpful when a summary is printed. So yes, you should always have a summary element under msContent. If there is text, then the given summary will be printed, if the element is empty, extracted summary will be produced. It is the TOC of the record.